cogs/moderation.py

```diff
@@ -654,6 +683,7 @@ async def tradingmute(
             target=target,
             user=ctx.author,
             reason=reason,
+            guild_id=ctx.guild.id,
```

Comment:
This *might* break on new servers? You have setup for muted but no setup or anythign for trading mute/trading

Reply:
its ok